docs/content/docs/deployment/elastic_scaling.md

```diff
@@ -152,7 +152,7 @@ The behavior of Adaptive Scheduler is configured by [all configuration options c
 
 ## Adaptive Batch Scheduler
 
-The Adaptive Batch Scheduler can automatically decide parallelisms of operators for batch jobs. If an operator is not set with a parallelism, the scheduler will decide parallelism for it according to the size of its consumed datasets. This can bring many benefits:
+The Adaptive Batch Scheduler is a batch job scheduler that can automatically adjust the execution plan and currently support automatically decide parallelisms of operators for batch jobs. If an operator is not set with a parallelism, the scheduler will decide parallelism for it according to the size of its consumed datasets. This can bring many benefits:
```

Comment:
plan and currently -> plan. It currently
support -> supports
decide -> deciding